xpublish/plugins/manage.py

```diff
@@ -0,0 +1,45 @@
+"""
+Load and configure Xpublish plugins from entry point group `xpublish.plugin`
+"""
+from importlib.metadata import entry_points
+from typing import Dict, Iterable, Optional
+
+from .hooks import Plugin
+
+
+def find_default_plugins(exclude_plugins: Optional[Iterable[str]] = None):
+    """Find Xpublish plugins from entry point group `xpublish.plugin`
+
+    Individual plugins may be ignored by adding them to `exclude_plugins`.
+    """
+    exclude_plugins = set(exclude_plugins or [])
+
+    plugins: Dict[str, Plugin] = {}
+
+    for entry_point in entry_points()['xpublish.plugin']:
+        if entry_point.name not in exclude_plugins:
+            plugins[entry_point.name] = entry_point.load()
+
+    return plugins
+
+
+def load_default_plugins(exclude_plugins: Optional[Iterable[str]] = None):
```

Comment:
return type?

Reply:
I'm adding return types for all of `plugins/manage.py` and cleaning up the types of everything in plugins.